src/taxpasta/infrastructure/application/metaphlan/metaphlan_profile_standardisation_service.py

```diff
@@ -83,31 +85,35 @@ def transform(
                 },
             )
         )
+        # MetaPhlAn sometimes assigns a taxon by name only, but no identifier exists.
+        # We have no choice but to count those entries as unclassified at the moment.
+        # We drop unclassified entries.
```

Comment:
I sort of feel we should warn for this if this happens (even if it's included in teh higher classified taxon), I'm not really happy with dropping 'valid' contents from the output of the tool when we just say we are essentially changing teh format...

Reply:
At a minimum in docs, but nice to have would be a warn on CLI

Reply:
There is a warning emitted if you look a few lines below.